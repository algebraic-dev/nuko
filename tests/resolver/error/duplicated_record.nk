type Ata = {
  b: Int,
  b: String
}