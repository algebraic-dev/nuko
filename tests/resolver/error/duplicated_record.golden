This AlreadyExistsName "b" FieldName
    ( Range
        { start = Pos
            { line = 2
            , column = 2
            }
        , end = Pos
            { line = 2
            , column = 3
            }
        }
    )
